script/check-collections.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+
+const defaultCollections = require('../script/collections/default.json');
+const brandConsolidationCollections = require('../script/collections/brand-consolidation.json');
+
+const checkCollections = buildOptions => (files, metalsmith, done) => {
+  const collection = buildOptions['brand-consolidation-enabled']
+    ? Object.keys(brandConsolidationCollections)
+    : Object.keys(defaultCollections);
+
+  const brokenCollections = Object.keys(files).reduce((accum, key) => {
+    const file = files[key];
+
+    if (key.includes('.html')) {
+      if (file.collection.length > 0) {
+        file.collection.forEach(collectionName => {
+          if (!collection.includes(collectionName)) {
+            accum.push(
+              `Error: Your [ collection "${collectionName}" ] does not exist in the collection: ${key}`,
+            );
+          }
+        });
+      }
```

Comment:
Is `file.collection` an array? I always thought it was just a single value.

Reply:
@ncksllvn Yes it's actually an array. It's because pages can have multiple collection items associated with them. I've seen it used on a couple pages.